src/main/java/eu/enmeshed/model/requestItems/RequestItemGroup.java

```diff
@@ -1,17 +1,27 @@
 package eu.enmeshed.model.requestItems;
 
+import static com.fasterxml.jackson.annotation.JsonInclude.Include.NON_EMPTY;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
 import java.util.List;
+import java.util.Map;
 import lombok.Getter;
 import lombok.NoArgsConstructor;
 import lombok.Setter;
 import lombok.experimental.SuperBuilder;
 
-// @AllArgsConstructor
 @NoArgsConstructor
 @Getter
 @Setter
 @SuperBuilder
+@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "@type")
+@JsonInclude(NON_EMPTY)
 public class RequestItemGroup extends RequestItem {
-
   private List<RequestItem> items;
+  private String title;
+  private String description;
+  private Map<String, String> metadata;
+  private Boolean mustBeAccepted;
```

Comment:
[as stated in our migration guide](https://enmeshed.eu/integrate/migration-from-v4-to-v5#removal-of-the-mustbeaccepted-property-of-the-requestitemgroup) `mustBeAccepted` was completely removed from the `RequestItemGroup`

Reply:
Removed